composer.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "friendsofcrawler/typo3-crawler-widget",
+  "type": "typo3-cms-extension",
+  "description": "Widgets for crawler extension",
+  "homepage": "https://github.com/tomasnorre/typo3-crawler-widget",
+  "support": {
+    "issues": "https://github.com/tomasnorre/typo3-crawler-widget/issues"
+  },
+  "license": "GPL-2.0-or-later",
+  "authors": [
+    {
+      "name": "Philipp Kuhlmay",
+      "email": "mail@treupo.de",
+      "homepage": "https://www.philippkuhlmay.de",
+      "role": "Developer"
+    }
+  ],
+  "require": {
+    "php": "^7.2",
+    "typo3/cms-core": "^10.4",
+    "typo3/cms-dashboard": "^10.4",
+    "aoepeople/crawler": "^9.0"
+  }
+  ,
+  "require-dev": {
+    "roave/security-advisories": "dev-master"
```

Comment:
I find it ok. But I see it more on the CI level of people setting it up. But it's not harming having it here. I would just do it differently.